governance-server/src/logic.js

```diff
@@ -142,7 +142,7 @@ module.exports = class Logic {
         const result = async () => {
             const pollState = await this.aeternity.pollState(address);
 
-            const votingAccounts = pollState.votes.map(([account, option]) => {
+            const votingAccounts = Object.entries(pollState.votes).map(([account, option]) => {
```

Comment:
the return value could be a bit more compact :)

Reply:
right now only making necessary changes not code-style refactor